tests/test_release_transpiler.py

```diff
@@ -32,13 +37,38 @@ def test_build_release_matrix(tmpdir):
         assert release_matrix["lib2"] == "2.3.4"
 
 
-def test_transpile(tmpdir):
+@pytest.mark.parametrize("coords", [({"py": ["3.8"], "rhel": ["7"]})])
```

Comment:
Do the tests produce the correct results if you use (say) `{"py": ["3.8", "3.10"], "rhel": ["7", "8"]}`?

Reply:
yeap, I included the suggestion in the test and updated the test data `test_release_matrix.yml` for that.